application/src/main/java/org/opentripplanner/service/vehicleparking/VehicleParkingService.java

```diff
@@ -0,0 +1,25 @@
+package org.opentripplanner.service.vehicleparking;
+
+import com.google.common.collect.ImmutableListMultimap;
+import java.util.stream.Stream;
+import org.opentripplanner.service.vehicleparking.model.VehicleParking;
+import org.opentripplanner.service.vehicleparking.model.VehicleParkingGroup;
+
+/**
+ * The read-only service for getting information about parking facilities.
+ * <p>
+ * For writing data see {@link VehicleParkingRepository}
+ */
+public interface VehicleParkingService {
+  Stream<VehicleParking> getBikeParks();
+
+  Stream<VehicleParking> getCarParks();
+
+  Stream<VehicleParking> getVehicleParkings();
+
+  ImmutableListMultimap<VehicleParkingGroup, VehicleParking> getVehicleParkingGroups();
```

Comment:
Fix:
 - No streams, 
 - use interface not implementation
 - naming

Reply:
Done in https://github.com/opentripplanner/OpenTripPlanner/pull/6246/commits/dd363aab68cacb173a5ada49a4bd888c19af329b